### ISSUE TITLE
How can we retrieve the confidence of each prediction made by the model?

### ISSUE THREAD

**Issue by kraakman** [naiveHobo/InvoiceNet#34]
I assume we need to extract it from the logits in the predict() function in invoicenet/acp/acp.py, but I cannot get it to work. Any help/suggestions are appreciated.

**Comment by niggle** [comment]
I already tried too but no luck

**Comment by kraakman** [comment]
@naiveHobo any suggestions on how to implement this?
